mm2src/coins/eth.rs

```diff
@@ -196,11 +222,39 @@ const GAS_PRICE_APPROXIMATION_PERCENT_ON_ORDER_ISSUE: u64 = 5;
 /// - it may increase by 3% during the swap.
 const GAS_PRICE_APPROXIMATION_PERCENT_ON_TRADE_PREIMAGE: u64 = 7;
 
-pub const ETH_GAS: u64 = 150_000;
+/// Heuristic gas limits for withdraw and swap operations (for swaps also including extra margin value for possible changes in opcodes gas)
+pub mod gas_limit {
+    /// Gas limit for sending coins
+    pub const ETH_SEND_COINS: u64 = 21_000;
+    /// Gas limit for transfer ERC20 tokens
+    /// TODO: maybe this is too much and 150K is okay
+    pub const ETH_SEND_ERC20: u64 = 210_000;
+    /// Gas limit for swap payment tx with coins
+    /// real values are approx 48,6K by etherscan
+    pub const ETH_PAYMENT: u64 = 65_000;
+    /// Gas limit for swap payment tx with ERC20 tokens
+    /// real values are 98,9K
+    pub const ERC20_PAYMENT: u64 = 120_000;
```

Comment:
we need to set this higher, see https://github.com/KomodoPlatform/komodo-defi-framework/issues/2135
maybe ERC20_RECEIVER_SPEND too

Reply:
this definitely should be tuned.
Should we maybe add different gas limit sets for different coins?
Or, maybe allow to override gas limit in coins file